javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+if (previousTodosJSON !== null) {
+  todos = JSON.parse(previousTodosJSON);
+}
+
+window.addEventListener('beforeunload', event => {
+  event.preventDefault();
```

Comment:
You don't need to use this here, because it won't have any impact. There's no "default" behavior we are trying to stop here, and since we are specifically using an unload event, it won't stop the page from refreshing